code/game/machinery/cloning.dm

```diff
@@ -284,32 +303,46 @@
 
 	H.suiciding = 0
 	attempting = 0
+
+	if(reagents && reagents.total_volume)

```

Comment:
The enzymatic reclaimer shoves all of it's reagents into cloning pods, not just biomatter.